spec/requests/paypal_confirm_spec.rb

```diff
@@ -52,6 +53,16 @@
       expect(order.adjustments.payment_fee.count).to eq 1
       expect(order.adjustments.payment_fee.first.amount).to eq 1.5
 
+      user = Spree::User.find(order.user_id)
+      expect(user).to_not be_nil
+
+      pp "WIP DEBUG:"
+      pp order.customer
+      pp order.customer.user
+      pp order
+
+      params[:token] = order.token
+
       get spree.confirm_paypal_path, params
```

Comment:
There's now a failing PayPal order confirmation spec that I can't debug. It fails in Travis but not on my local machine. I've added some temporary lines into the spec to see what's happening but still can't figure it out. The order in the test has `user_id=1449`, but then when the order status gets updated for some reason it's trying to save the order with `user_id=1447`, which doesn't exist.

https://travis-ci.org/openfoodfoundation/openfoodnetwork/jobs/284155732#L3412

Reply:
I've tried setting the user explicitly and tweaking the params as well.

Reply:
That spec failed in several pull requests now, but the culprit was actually in a different spec changing the session. Rob just fixed it: https://github.com/openfoodfoundation/openfoodnetwork/pull/1898